lib/screens/app/wallet/dashboard.dart

```diff
@@ -361,76 +359,18 @@ class _DashboardState extends State<Dashboard> {
-                            ),
-                            TransactionAvatar(
-                              size: 40,
-                              account: member.data.account,
-                              nickname: member.data.nickname,
-                              image: member.data.image,
-                              decoration: BoxDecoration(
-                                shape: BoxShape.circle,
-                                color: AppColors.blue,
-                              ),
-                            ),
-                            Flexible(
-                                child: Container(
-                                    margin: EdgeInsets.only(left: 10, right: 10),
-                                    child: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
-                                      Container(
-                                        child: Text(
-                                          member.data.nickname,
-                                          maxLines: 1,
-                                          style: TextStyle(fontWeight: FontWeight.w600, fontSize: 15),
-                                        ),
-                                      ),
-                                      Container(
-                                        child: Text(
-                                          member.data.account,
-                                          maxLines: 1,
-                                          style: TextStyle(color: AppColors.grey, fontSize: 13),
-                                        ),
-                                      ),
-                                    ])))
-                          ],
-                        )),
-                        Container(
-                            margin: EdgeInsets.only(left: 10, right: 15),
-                            child: Row(
-                              children: <Widget>[
-                                Text(
-                                  type == TransactionType.income ? '+' : '-',
-                                  style: TextStyle(
-                                      color: type == TransactionType.income ? AppColors.green : AppColors.red,
-                                      fontSize: 16),
-                                ),
-                                Text(
-                                  model.quantity.seedsFormatted + " SEEDS",
-                                  style: TextStyle(fontWeight: FontWeight.w600, fontSize: 15),
-                                )
-                              ],
-                            ))
-                      ],
-                    ),
-                  ),
-                ],
-              ),
+          ? TransactionInfoCard(
+              callback: () {
+                onTransaction(transaction: model, member: member.data, type: type);
+              },
+              transactionProfileAccount: member.data.account,
```

Comment:
remove the `Transaction` from these field names. 
profileAccount
profileNickname

Etc

Reply:
Fixed, I added so it would be more clear, for example "Profile account" is from the transaction profile account not from the user profile account.